### DISCUSSION TITLE
Deaktivierung GPS Funktion wenn Enhanced Vehicle aktiv ist

### DISCUSSION THREAD

**Discussion by 0scarZulu** [Mogli12/KeyboardSteer#762] [Ideas]
Hallo, besteht die Möglichkeit der deaktivierung der GPS Funktion wenn Enhanced Vehicle aktiv im Modordner ist? 

Noch bessere wäre eine Möglichkeit der manuellen Aktivierung oder Deaktivierung der  Spurführung.

Beste Grüße

**Comment by Mogli12** [comment]
Hallo,

Am besten deaktivierst du die Tastenbelegungen von VCA. Dann ist die Spurführung aus und bleibt auch aus.

Viele Grüße

**Reply by 0scarZulu** [reply]
Hm ja ok und was is mit den überlappenden Zahlen über dem Tacho :)

**Comment by Mogli12** [comment]
Ich habe mir mal https://github.com/ZhooL/FS22_EnhancedVehicle/issues/61 angeschaut. Wenn ZhooL das so sieht, dann kann man wohl nur entweder VCA oder EV verwenden.

**Comment by 0scarZulu** [comment]
Also lehnst du von deiner Seite eine Funktion der Deaktivierung übers Menü oder wie bei Guidance sobald das aktiv ist ab .. Ok schade

**Comment by Mogli12** [comment]
Ich kann leider keine ordentlich benannte Variable wie bei Guidance Steering finden. Auf Basis von self.vData[5] werde ich keine Aktionen in VCA vornehmen.

**Comment by Kynuska** [comment]
Ich glaube, er meint, in einem Laden oder einer Werkstatt einen Mod für eine Maschine zu kaufen und ihn damit zu (de)aktivieren.

**Comment by 0scarZulu** [comment]
Ja genau entweder so. Oder das man es über das Menü von VCA deaktiviert. So wie die Sperren zB

Edit: Oder über die Modsettings? Blei darüber steuerbar machen?

**Comment by Mogli12** [comment]
Ich werde in VCA keinen Schalter einbauen um Kompatibilität mit einem anderen Mod herzustellen, wenn der Autor des anderen Mods schreibt: "I don't know and to be honest I don't care."

**Comment by 0scarZulu** [comment]
Ok wo schreibt er das? 
Hatte mit ihm eigentlich immer guten Kontakt auf Discord. Und englisch? Er ist doch auch deutsch

**Comment by Mogli12** [comment]
Der Link steht oben

**Comment by ZhooL** [comment]
Ganz einfach: ich habe keine Lust auf den Kindergarten, den wir im LS19 schon hatten. Ich respektiere Moglis und Wopsters Arbeit und daher können bei EV alle Funktionen und Anzeigen etc. selbständig im Menü ausgeschaltet werden und/oder mit etwas Hirn per Config XML verschoben werden um Platz für andere Mods Anzeigen zu machen. Ich werde jedoch nicht mehr im Code prüfen ob andere Mods geladen sind und dadurch automatisch Funktionen/Elemente (de)aktivieren/verschieben oder schlimmer noch in den Code von anderen Mods eingreifen ("I don't care"). Das Gleiche würde ich mir auch von anderen Mods wünschen... Platz rund um das Default HUD und Tastenbelegung ist leider begrenzt.
Ich habe VCA nicht installiert; ich sage anderen "Nutze VCA oder nutze EV - beides parallel klingt nicht sinnvoll"; aus der Community höre ich, dass beide parallel installiert keine Probleme machen, solange natürlich nicht beide die gleichen Funktionen (z.B. Spurhalten) aktiv haben.

**Reply by manuelgbel** [reply]
Welche Werte müsste man den ändern um ein Hub zu verschieben? Wenn ich mal fragen darf :)
Ist die guiProfiles-Datei oder?

**Reply by ZhooL** [reply]
In der modSettings\FS22_EnhancedVehicle\FS22_EnhancedVehicle_v1.xml gibt es im Block <hud> diverse "offsetX" und "offsetY" Parameter, mit denen einige Elemente verschoben werden können. Entweder anpassen wenn das Spiel nicht läuft, oder während es läuft anpassen und anschließend im EV Menü "XML neu laden" Button drücken. Oder im EV Menü HUD Elemente komplett deaktivieren, wenn du sie nicht brauchst.

**Reply by manuelgbel** [reply]
Hi, danke für die Antwort. Ich würde aber gerne die Huds von VCA verschieben. Wo geht es dort? Diese Modsetting-Datei ist nämlich leer

**Reply by RysingTV** [reply]
> Hi, danke für die Antwort. Ich würde aber gerne die Huds von VCA verschieben. Wo geht es dort? Diese Modsetting-Datei ist nämlich leer



Dann wirst du damit leben müssen.

**Comment by schuelli238** [comment]
Ich denke wenn grundsätzlich alle (nicht nur eure beiden) Mods einfach die Möglichkeit geben würden on/off einzelner Bereiche per Menu zu steuern + Optik noch mal per xml zu fein zu tunen, wären alle super glücklich. Ich zumindest :-)

Ob sich die Mods untereinander beißen oder stören, das sollte nicht euer Problem und muss nachher jeder User für sich entscheiden, dafür deckt ihr einfach zu viel schon gegenseitig ab :-) ... Viele Grüße und danke für beide Mods. I love them.

**Comment by Mogli12** [comment]
Nochmal: Wenn VCA nur deshalb verwendet wird, um die Ganggeschwindigkeit unter dem Tacho anzuzeigen, dann wäre es doch einfacher, wenn @ZhooL das ähnlich in Enhanced Vehicle einbaut, oder?

**Comment by 0scarZulu** [comment]
Ich nutze nun beides und warum auch immer überlappen die Zahlen nicht mehr 😂 die von vca ist einfach gar nicht erst da .. naja mit egal .. ich kann beide nutzen inkl. Vorteile und gut ist
